ExampleMod/Content/Items/Weapons/ExampleCloneWeapon.cs

```diff
@@ -0,0 +1,49 @@
+using ExampleMod.Content.Projectiles;
+using Terraria.ID;
+using Terraria.ModLoader;
+using static Terraria.ModLoader.ModContent;
+
+namespace ExampleMod.Content.Items.Weapons
+{
+	/// <summary>
+	///		This weapon and its corresponding projectile showcase the CloneDefaults() method, which allows for cloning of other items.
+	///		For this example, we shall copy the Meowmere and it's projectiles with the CloneDefaults() method, while also changing
+	///		them slightly.
+	///		For a more detailed description of each item field used here, check out <see cref="ExampleSword" />.
+	/// </summary>
+	public class ExampleCloneWeapon : ModItem
+	{
+		public override void SetStaticDefaults() {
+			DisplayName.SetDefault("Meowmere V2");
+		}
+
+		public override void SetDefaults() {
+			//This method right here is the backbone of what we're doing here; by using this method, we copy all of
+			//the meowmere's SetDefault stats (such as item.melee and item.shoot) on to our item, so we don't have to
+			//go into the source and copy the stats ourselves. It saves a lot of time and looks much cleaner; if you're
+			//going to copy the stats of an item, use CloneDefaults().
+			
+			item.CloneDefaults(ItemID.Meowmere);
+
+			//After CloneDefaults has been called, we can now modify the stats to our wishes, or keep them as they are.
+			//For the sake of example, lets swap the vanilla Meowmere projectile shot from our item for own projectile by changing item.shoot:
+			
+			item.shoot = ProjectileType<ExampleCloneProjectile>(); //Remember we must used ProjectileType<>() since it is a modded projectile!
```

Comment:
```suggestion
			item.shoot = ProjectileType<ExampleCloneProjectile>(); //Remember that we must use ProjectileType<>() since it is a modded projectile!
```